packages/vite/src/node/plugins/css.ts

```diff
@@ -279,7 +283,10 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
         return
       }
 
+      const query = parseRequest(id)
       const inlined = inlineRE.test(id)
+      const inlineCSS = inlineCSSRE.test(id)
+      const isHTMLProxy = htmlProxyRE.test(id)
```

Comment:
If we move these regex-tests down a bit, we could potentially safe some nanoseconds 😛